Stackage/PackageIndex.hs

```diff
@@ -110,10 +109,6 @@ data SimplifiedPackageDescription = SimplifiedPackageDescription
     }
     deriving Generic
 
-#if !MIN_VERSION_base(4, 9, 0)
```

Comment:
This should be the test of cabal version `!MIN_VESION_Cabal(2,0,0)`, but stackage-curator dropped Cabal <2 support